tokens.xml

```diff
@@ -6837,6 +6852,21 @@ Creatures you control attack each combat if able.</text>
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Ravage</name>
+            <text>Menace, deathtouch</text>
+            <prop>
+            	<colors>B</colors>
+                <type>Token Legendary Artifact Creature — Robot</type>
+                <maintype>Creature</maintype>
+                <cmc>0</cmc>
+                <pt>3/3</pt>
+            </prop>
+            <set picURL="https://cards.scryfall.io/large/front/b/3/b3170b92-b4eb-4826-92ed-85deba4c4c88.jpg?1666063013">BOT</set>
+            <reverse-related>Soundwave, Superior Captain</reverse-related>
```

Comment:
```suggestion
            <reverse-related exclude="exclude">Soundwave, Superior Captain</reverse-related>
```
with multiple relations you need to exclude one, otherwise the default keybind for create token (ctrl shift t) will make all at the same time.